Examples/ExampleTextToSpeechV1.cs

```diff
@@ -61,12 +68,57 @@ private void PlayClip(AudioClip clip)
         }
         #endregion
 
+        #region Concatenate Byte Arrays
```

Comment:
Can we move these util functions to the bottom of the file? Both PlayClip and ConcatenateByteArrays